sections/04_study.md

```diff
@@ -525,9 +525,11 @@ types where the subset percentage ranged from 0.1 to 1%, significantly
 outperforming logistic regression and distance-based outlier detection methods.
 However, they did not benchmark against random forests, which tend to work
 better for imbalanced data, and their data was relatively low dimensional.
-Future work is needed to establish the utility of deep learning in cell subset
-identification, but the stunning improvements in image classification over the
-past 5 years [@tag:He2015_images] suggest transformative potential.
+
+Neural networks can also learn meaningful low dimensional representations of single cell expression data [@doi:10.1093/nar/gkx681 @doi:10.1101/178624], which can be used to identify cellular subpopulations or match new cells to known cell types.
```

Comment:
I read the [Ding et al.](http://www.biorxiv.org/content/early/2017/09/01/178624) paper yesterday. The paper is about a deep VAE trained to embed a 2 dimensional hidden layer. It's used primarily as a visualization tool that the authors demonstrate out-compete PCA/t-SNE in 1) preserving local distances in the presence of outlier cells and 2) projecting new data into 2D space. They do not go into much detail about identifying subpopulations or matching known cell types. I suggest reworking the citation to this paper

Reply:
For reference, [Lopez et al.](https://arxiv.org/abs/1710.05086) also examines the use of VAEs for scRNA-seq, but does more analysis than just visualization. It considers specific use cases (e.g., differential expression, imputing missing data) as well.

Reply:
@evancofer thanks, I hadn't seen that one.  I'd like to merge this pull request soon so I'll check it out.  I created #682.